bin/utils/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
I'm not sure if this file should be in `bin` dir since it is not executable as is. Maybe we should move this to `lib`?

Reply:
I will address this in the PR I send for refactoring the codebase